EKF/mag_fusion.cpp

```diff
@@ -623,14 +623,14 @@ void Ekf::fuseHeading()
 		}
 	}
 
-	// wrap the heading to the interval between +-pi
-	measured_hdg = wrap_pi(measured_hdg);
```

Comment:
I have modelled both the legacy wrap_pi  https://github.com/PX4/Matrix/blob/cf924956d7d62ce18bfc4f8441e9177ddb69c0dc/matrix/helper_functions.hpp#L21-L38) and the algorithm from this PR  in Matlab here: 

[test.txt](https://github.com/PX4/ecl/files/1164500/test.txt)

Both are producing the same output so I do not see the point of this change.